### PR TITLE
Replaced ProfileCard with CollectiveCard component.

### PR DESCRIPTION
**Pull request by carlosascari**
A branch with the same name exists in OpenCollective/api repository, it provides the data that a CollectiveCard expects: https://github.com/OpenCollective/opencollective-api/pull/470
The API branch **MUST** be merged first.

------------------------
@asood123 Here is a screenshot of the CollectiveCard: 
On the left is a card with default logo and background.
On the right is a card with yeoman's data filled in.

preview

### CODE REVIEW

**Comment by asood123** [comment]
Can you share what this component looks like with real data?

**Comment by asood123** [comment]
@carlosascari the card looks nice.

**Comment by asood123** [comment]
I'll wait to merge this after the API branch is merged.

File: frontend/src/css/main.css [hunk 1]
```diff
@@ -80,7 +80,7 @@
 @import './components/UserPhoto.css';
 @import './components/Widget.css';
 @import './components/YoutubeVideo.css';
-
+@import './components/CollectiveCard.css';
```
Comment (asood123) [inline frontend/src/css/main.css]: In the future, can you keep these alphabetical?

Comment (carlosascari) [inline frontend/src/css/main.css]: I will

**Pull request merged**